src/expression/parse.js

```diff
@@ -788,15 +792,17 @@ export const createParse = /* #__PURE__ */ factory(name, dependencies, ({
 
   /**
    * logical and, 'x and y'
+   * logical nand, 'x nand y'
    * @return {Node} node
```

Comment:
As I mentioned in earlier review, `nand` is actually a form of `or` (since `A nand B == (not A) or (not B)`). So `nand` should go in the logical or precedence group (maybe rename it parseLogicalOrNand?? that's optional). And conversely `A nor B == (not A) and (not B)` is a form of `and` so should go in this precedence group, which you could optionally rename `parseLogicalAndNor`. Thanks!